docs/en/relying-party-solution.rst

```diff
@@ -477,21 +455,42 @@ Where the following parameters are used:
   * - **Name**
     - **Description**
   * - **vp**
-    - The digital credential in its original state. The public key contained in the digital credential MUST be used to verify the entire VP JWS as Proof of Possession of the private key which the public part is carried in the digital credential (Holder Key Binding).
+    - The Digital Credential in its original state. The public key contained in the Digital Credential MUST be used to verify the entire VP JWS as Proof of Possession of the private key which the public key is included in the Digital Credential. Eg: for SD-JWT VC the pblic key is provided within the ``cnf.jwk`` claim.
   * - **jti**
-    - JWS Unique identifier.
+    - JWS unique identifier.
   * - **iat**
-    - Unix timestamp of the issuance datetime.
+    - Unix timestamp of the time of issuance of this presentation.
   * - **exp**
-    - Unix timestamp beyond which the presentation of the digital credential will no longer be considered valid.
+    - Unix timestamp beyond which this presentation will no longer be considered valid.
   * - **aud**
-    - Audience of the VP, corresponding to the ``redirect_uri`` within the Authorization request issued by the Relying Party.
+    - Audience of the VP, corresponding to the ``response_uri`` within the Authorization request issued by the Relying Party.
   * - **nonce**
-    - Nonce provided by the Relying Party within the Authorization Request.
+    - The nonce value provided by the Relying Party within the Authorization Request.
+
+
+Redirect URI
+------------
+
+When the Relying Party provides the redirect URI, the Wallet MUST send the user-agent to this redirect URI. The redirect URI allows the Relying Party to continue the interaction with the End-User on the device where the Wallet resides after the Wallet has sent the Authorization Response to the response URI.
+
+The Relying Party MUST include a response code withing the redirect URI. The response code is a fresh, cryptographically random number used to ensure only the receiver of the redirect can fetch and process the Authorization Response. The number could be added as a path component, as a parameter or as a fragment to the URL. It is RECOMMENDED to use a cryptographic random value of 128 bits or more at the time of the writing of this specification.
+
+The following is a non-normative example of the response from the Relying Party  to the Wallet upon receiving the Authorization Response at the Response Endpoint.
+
+
+.. code-block:: http
+
+  HTTP/1.1 200 OK
+  Content-Type: application/json;charset=UTF-8
+
+  {
+    "redirect_uri": "https://relying-party.example.org/cb#response_code=091535f699ea575c7937fa5f0f454aee"
+  }
+
```

Comment:
what is the response obtained when the Wallet Instance makes a call to the `redirect_uri`?

Reply:
good catch, I'll provide guidance here

Reply:
Done!